docs/tasks/federation/Policy-Engine-Deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
```

Comment:
Will fix. Was copying the title case Values.yaml (which may just be a helm thing?)